src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
oops, you need to keep modifiers functional,
so if i change the class to `stars stars--3`:
<img width="732" alt="Monosnap Product cards 2022-11-08 17-30-12" src="https://user-images.githubusercontent.com/39308717/200606985-1a669d42-86b6-4e6b-a2af-bd035e10c8b2.png">

Reply:
just use all the styling from stars task